storage/local/persistence_test.go

```diff
@@ -884,3 +884,74 @@ func verifyIndexedState(i int, t *testing.T, b incrementalBatch, indexedFpsToMet
 		}
 	}
 }
+
+var fpStrings = []string{
+	"b004b821ca50ba26",
+	"b037c21e884e4fc5",
+	"b037de1e884e5469",
+}
+
+func BenchmarkLoadChunksSequentially(b *testing.B) {
+	p := persistence{
+		basePath: "fixtures",
+	}
+	sequentialIndexes := make([]int, 47)
+	for i := range sequentialIndexes {
+		sequentialIndexes[i] = i
+	}
+
+	var fp clientmodel.Fingerprint
+	for i := 0; i < b.N; i++ {
+		for _, s := range fpStrings {
+			fp.LoadFromString(s)
+			cds, err := p.loadChunks(fp, sequentialIndexes, 0)
+			if err != nil {
+				b.Error(err)
+			}
+			if len(cds) == 0 {
+				b.Error("could not read any chunks")
+			}
+		}
+	}
+}
+
+func BenchmarkLoadChunksRandomly(b *testing.B) {
+	p := persistence{
+		basePath: "fixtures",
+	}
+	randomIndexes := []int{1, 5, 6, 8, 11, 14, 18, 23, 29, 33, 42, 46}
```

Comment:
Would one not want to load all 47 chunks here too (randomly), unless you don't care about comparing it to the benchmark above?